src/metrics/prometheus_histogram.erl

```diff
@@ -95,7 +95,7 @@ the number of time ticks when the recent value was observed).
 -define(ISUM_POS, 3).
 -define(FSUM_POS, 4).
 -define(BUCKETS_START, 5).
--define(WIDTH, 16).
+-define(WIDTH, erlang:system_info(schedulers_online)).
```

Comment:
Maybe instead of fixing this to be `schedulers_online` we can set this up as an `application:get_env(<application>, <some_name>, 16)` so the original remains preserved but we open up the loophole for setting a higher value.

Reply:
This solution has the same issue as @mikpe mentioned above, we don't want to let users change that value dynamically. another issue with defaults - they are rarely changed :)